presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -667,6 +668,8 @@ private void createHiveTable(ConnectorSession session, ConnectorTableMetadata ta
 
         tableProperties.putAll(createZeroStatistics().toPartitionParameters());
 
+        tableProperties.put(NUM_PARTITIONS, "0");
```

Comment:
Should we set this property for non-partitioned tables?

Reply:
No, good point